tools/admin/requests/physician-assessment.ts

```diff
@@ -10,3 +14,27 @@ export type PhysicianAssessment = Pick<
   patientCondition: PatientCondition | null;
   permitType: PermitType | null;
 };
+
+/** Get physician assessment information of an application */
+export const GET_PHYSICIAN_ASSESSMENT = gql`
+  query GetPhysicianAssessment($id: Int!) {
+    application(id: $id) {
+      id
+      __typename
+      ... on NewApplication {
+        disability
+        disabilityCertificationDate
+        otherPatientCondition
+        temporaryPermitExpiry
+        patientCondition
+        permitType
+      }
+    }
+  }
+`;
+
+export type GetPhysicianAssessmentRequest = QueryApplicationArgs;
+
+export type GetPhysicianAssessmentResponse = {
+  application: PhysicianAssessment;
```

Comment:
Please `Pick` fields here so that in case any schemas change in the future, we don't get TS errors

Reply:
Addressed! I don't fully grasp the difference between using `PhysicianAssessment` (which also picks fields) vs. picking the fields again. If you have the time to explain that would be appreciated!

Reply:
Sounds good, I can explain offline!